test/component/uk/gov/hmrc/exports/ExportsSubmissionReceivedSpec.scala

```diff
@@ -17,47 +17,54 @@
 package component.uk.gov.hmrc.exports
 
 import component.uk.gov.hmrc.exports.base.ComponentTestSpec
-import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
 import play.api.libs.json.Json
-import play.api.mvc.{AnyContentAsJson, Result}
+import play.api.mvc.{AnyContent, AnyContentAsJson, Result}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 import uk.gov.hmrc.exports.models.Choice
 import uk.gov.hmrc.exports.models.declaration.ExportsDeclaration.REST.format
-import uk.gov.hmrc.exports.models.declaration.{Seal, TransportInformationContainer}
+import uk.gov.hmrc.exports.models.declaration.{ExportsDeclaration, Seal, TransportInformationContainer}
 import uk.gov.hmrc.http.InternalServerException
 import util.CustomsDeclarationsAPIConfig
 import util.testdata.ExportsDeclarationBuilder
 import util.testdata.ExportsTestData._
 
 import scala.concurrent.Future
 
-class ExportsSubmissionReceivedSpec extends ComponentTestSpec with ScalaFutures with ExportsDeclarationBuilder {
+class ExportsSubmissionReceivedSpec extends ComponentTestSpec with ScalaFutures with ExportsDeclarationBuilder with IntegrationPatience {
 
-  lazy val ValidSubmissionRequest: FakeRequest[AnyContentAsJson] = FakeRequest("POST", endpoint)
+  private val declaration: ExportsDeclaration = aDeclaration(
+    withChoice(Choice.StandardDec),
+    withId("id"),
+    withEori(declarantEoriValue),
+    withConsignmentReferences(lrn = declarantLrnValue),
+    withContainerData(TransportInformationContainer("container123", Seq(Seal("seal1"))))
+  )
+
+
+  override protected def beforeEach(): Unit = {
```

Comment:
This is not necessary, I think in places where we don't need to override beforeEach or afterEach, because we don't need to add anything there we shouldn't put it